tools/phenomenal/nmr/bruker2batman.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--Proposed Tool Section: [File Handling]-->
+<tool id="bruker2batman" name="bruker2batman Converter">
+     <description>Bruker NMR files to BATMAN converter</description>
+     <requirements>
+       <container type="docker">container-registry.phenomenal-h2020.eu/phnmnl/bruker2batman</container>
+     </requirements>
+     <command><![CDATA[   
+     export indir=`dirname $BrukerNMRzip`;
```

Comment:
This won't work as $BrukerNMRzip will be in a directory that is not the Galaxy working directory for the job. Simply unzip the contents of BrukerNMRZip in the working directory `\$PWD` and then issue `bruker2batman.R -i \$PWD` instead. That should work with the outputs that you have set below.